.changeset/afraid-guests-jog.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/livechat": minor
+---
+
+Created a transferChat Livechat API endpoint for transferring chats programmatically, the endpoint has all the limitations & permissions required that transferring via UI has
```

Comment:
```suggestion
Created a `transferChat` Livechat API endpoint for transferring chats programmatically, the endpoint has all the limitations & permissions required that transferring via UI has
```

One thing: should we call these `api endpoints`? 🤔 could be like `api hook`? But if for the UI makes sense to have it as endpoint pls ignore

Reply:
We were calling it endpoints cause we use them to access the inside of the iframe context, i do agree that is confusing, maybe we should have a more broad discussion outside this PR

Reply:
That's fair 👀 yeah, a bit confusing.

btw, you can add the backticks to the `transferChat` so it looks like code on release 🙏🏽